assets/js/protocol/protocol-navigation.js

```diff
@@ -175,13 +177,46 @@ if (typeof window.Mzp === 'undefined') { // eslint-disable-line block-scoped-var
             }
         }
     };
+    /**
+     * use Intersection Observer API to keep track of when the mobile
+     * nav menu is displayed to handle aria roles better
+     */
+    Navigation.menuButtonVisible = function (callback) {
+        // check if Intersection observer is supported
+        if (Mzp.Supports !== 'undefined' && Mzp.Supports.intersectionRatio) {
```

Comment:
Question: should this be `intersectionObserver`?
```suggestion
        if (Mzp.Supports !== 'undefined' && Mzp.Supports.intersectionObserver) {
```